src/main/resources/application.yaml

```diff
@@ -12,3 +12,11 @@ management:
 spring:
   application:
     name: Bulk Scan CCD Event Handler Sample Application
+
+idam:
+  s2s-auth:
+    url: ${S2S_URL:http://localhost:4552}
+    secret: ${S2S_SECRET:AAAAAAAAAAAAAAAA}
+    name: ${S2S_NAME:bulk_scan}
+
+allowed-services: bulk_scan_processor,bulk_scan_processor_test
```

Comment:
again, list change missing

Reply:
It's not missing this time 🙂 
Yaml list property is not working with `@Value` annotation. To keep it simple, I added it as comma-separated values.

Reply:
done